packages/kcms/src/team/adaptor/validator/team.ts

```diff
@@ -35,3 +35,13 @@ export const PostTeamsRequestSchema = z.array(
 );
 
 export const PostTeamsResponseSchema = z.array(TeamSchema).openapi('Teams');
+
+export const TeamIdParamsSchema = z.object({
```

Comment:
他のスキーマと命名を揃えて、
```suggestion
export const DeleteTeamParamsSchema = z.object({
```
としてほしいです。